build/readme-parser.ts

```diff
@@ -36,7 +36,7 @@ function extractDataFromMatch(match: RegExpMatchArray): FeatureMeta {
 	return {
 		id: simpleId as FeatureID,
 		description: parseMarkdown(linkLessMarkdownDescription),
-		screenshot: urls.find(url => /\.(png|gif|jpg|mp4|mov)$/.test(url)),
+		screenshot: urls.find(url => /\.(png|gif)$/i.test(url)),
```

Comment:
How about
```suggestion
		screenshot: urls.find(url => url.endsWith('.png') || url.endsWith('.gif')),
```
?

Reply:
I think the plain regex works well here

Reply:
We need to either add `jpg` back or replace the JPG screenshots.